providers/common/digests/sha3_prov.c

```diff
@@ -228,7 +242,11 @@ const OSSL_DISPATCH name##_functions[] = {                              \
     { OSSL_FUNC_DIGEST_FREECTX, (void (*)(void))keccak_freectx },       \
     { OSSL_FUNC_DIGEST_DUPCTX, (void (*)(void))keccak_dupctx },         \
     { OSSL_FUNC_DIGEST_GET_PARAMS, (void (*)(void))name##_get_params }, \
+    { OSSL_FUNC_DIGEST_GETTABLE_TYPES,                                  \
```

Comment:
Not sure there is a great correlation between the names OSSL_FUNC_DIGEST_GET_PARAMS & 
OSSL_FUNC_DIGEST_GETTABLE_TYPES. This is the type of the params that can be used by OSSL_FUNC_DIGEST_GET_PARAMS. It should at least have param in the name.